File: SIMS/src/main/java/uph/ii/SIMS/DocumentModule/Dto/OswiadczenieDto.java
package uph.ii.SIMS.DocumentModule.Dto;

import lombok.*;

import javax.persistence.Lob;

/**
 * Klasa, zawierająca wszystkie dane potrzebne do utworzenia nowego dokumentu oswiadczenie
 */
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@EqualsAndHashCode
public class OswiadczenieDto {
    private Long id;
    private Long groupId;
    private Long ownerId;
    private String opiekunI = "";
    private String opiekunN = "";
    private String opiekunMail = "";
    private String opiekunTel = "";
    @Lob
    private String studentDuties = "";
    private String comment = "";
    private String status = "";
    
    public OswiadczenieDto(Long id, Long groupId, Long ownerId) {
        this.id = id;
        this.groupId = groupId;
        this.ownerId = ownerId;
    }
}

File: SIMS-Front/src/app/oswiadczenie/oswiadczenieDto.ts

export class OswiadczenieDto {
  private id: number = null;
  private opiekunI: string;
  private opiekunN: string;
  private opiekunMail: string;
  private opiekunTel: string;


  constructor(id: number = null, opiekunI: string, opiekunN: string, opiekunMail: string, opiekunTel: string) {
    this.id = id;
    this.opiekunI = opiekunI;
    this.opiekunN = opiekunN;
    this.opiekunMail = opiekunMail;
    this.opiekunTel = opiekunTel;
  }

}
